frontend/src/components/activity/ScheduleEntry.vue

```diff
@@ -196,11 +212,33 @@ Displays a single scheduleEntry
                     {{ dateShort(scheduleEntryItem.start) }}
                   </td>
                   <td class="text-left tabular-nums pb-2 pr-0">
-                    {{ rangeLongEnd(scheduleEntryItem.start, scheduleEntryItem.end) }}
+                    <RouterLink
+                      v-if="
+                        category.numberingStyle === '-' &&
+                        scheduleEntryItem._meta.self !== scheduleEntry._meta.self
+                      "
+                      :to="scheduleEntryRoute(scheduleEntryItem)"
+                      class="e-title-link"
+                    >
+                      {{ rangeLongEnd(scheduleEntryItem.start, scheduleEntryItem.end) }}
+                    </RouterLink>
+                    <template v-else>
+                      {{ rangeLongEnd(scheduleEntryItem.start, scheduleEntryItem.end) }}
+                    </template>
                   </td>
                 </tr>
               </tbody>
             </table>
+            <DialogActivityEdit
+              v-if="activity && isContributor"
```

Comment:
shouldn't scheduleEntry be here, when its used as a prop?

Reply:
fixed